acceptance/bundle/artifacts/artifact_path_with_volume/volume_not_deployed/test.toml

```diff
@@ -0,0 +1,3 @@
+Badness = "Requires two bundles; one for volumes, another for artifacts. Should just work"
```

Comment:
I find the note "Should just work" too ambiguous and not helping the reader. Could you please elaborate on what you expect the CLI should do in this case? e.g. it should figure out the right order of deployment and deploy the volumes first and then upload artifacts?

Reply:
I don't know what the solution is at the moment. Looking at the config I don't see why it cannot "just work" but maybe there are some fundamental blockers. It's not in scope of this PR to figure that out though.

Reply:
Currently in DABs we upload artifacts before we do TF apply thus volume used for artifacts path is not yet created and hence not working. We (potentially) could upload artifacts after TF apply but I'm not 100% sure if there are existing blockers for it

Reply:
Thanks -- added this context to the message 5ffd06345d8546edc22fba0f583f5349f5436543